embedded-nal-async/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "embedded-nal-async"
+version = "0.1.0" # remember to update html_root_url
+authors = [
+    "Jonathan 'theJPster' Pallant <github@thejpster.org.uk>",
+    "Mathias Koch <mk@blackbird.online>",
+    "Diego Barrios Romero <eldruin@gmail.com>",
+    "Ryan Summers <ryan.summers@vertigo-designs.com>",
+    "Ulf Lilleengen <lulf@redhat.com>",
+]
+edition = "2018"
+description = "An Async Network Abstraction Layer (NAL) for Embedded Systems"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rust-embedded-community/embedded-nal"
+homepage = "https://github.com/rust-embedded-community/embedded-nal"
```

Comment:
Can you fix the links? Should have `/embedded-nal/async` appended

Reply:
Hmm, referring to a folder within a repository would require pointing to a specific branch, which seems a bit brittle to me.

Reply:
Hm, pointing to the root is likely fine. Users will be able to obviously find the async variant.